src/tool/cppwinrt/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.9)
+
+set(build msbuild /nologo /m /p:Configuration=${CMAKE_BUILD_TYPE},Platform=$ENV{VSCMD_ARG_TGT_ARCH} /p:CppWinRTVersion=${XLANG_BUILD_VERSION} ${CMAKE_CURRENT_SOURCE_DIR}/cppwinrt.sln)
```

Comment:
this is where you could pass something like:
/p:OutDir=${CMAKE_CURRENT_BINARY_DIR}\build
/p:IntDir=${CMAKE_CURRENT_BINARY_DIR}\build\temp\prebuild

Reply:
(and remove/default these in all vcxproj files)

Reply:
Ah, perfect.

Reply:
you could also have a vcxproj default that happens to align with what cmake passes in (or simply use that default all the time).  See here the CmakeBinaryDir property here for an example: https://github.com/Microsoft/xlang/blob/scottj1s/javax/src/tool/java/javart/vs/javart.vcxproj

Reply:
done